include/seqan3/argument_parser/validators.hpp

```diff
@@ -274,11 +274,12 @@ class value_list_validator
  * \{
  */
 //!\brief Deduction guide for a parameter pack over an arithmetic type.
-template <arithmetic ...option_types>
+template <typename option_type, typename ...option_types>
 //!\cond
-    requires (!(detail::is_char_adaptation_v<option_types> || ...))
+    requires ((arithmetic<option_types> && ... && arithmetic<option_type>) &&
```

Comment:
don't you wanna use `std::is_arithmetic` instead of `seqan3::arithmetic` right away?

Reply:
I changed that in a different PR :)